docs/avatar.html

```diff
@@ -1 +1,93 @@
+s<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="/styles/main-style.css">
+    <link rel="stylesheet"
+      href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/default.min.css">
+</head>
+<body>
+    <nav class="navigation-bar">
+        <div class="navigation-logo"><img src="/Images/main logo.png" alt="" ></div>
+        <div class="navigation-text">
+            <div class="text-one">Docs</div>
+            <div class="text-two">Support</div>
+        </div>
+    </nav>
 
+    <aside class="sidebar">
+        <h2>Components</h2>
+        <ul>
+           <li><a href="avatar.html">Avatar</a></li> 
+            <li><a href="alert.html">Alert</a></li>
+            <li><a href="badge.html">Badge</a></li>
+            <li><a href="button.html">Button</a></li>
+            <li><a href="card.html">Card</a></li>
+            <li><a href="image.html">Image</a></li>
+            <li><a href="input.html">Input</a></li>
+            <li><a href="text-utilities.html">Text Utilities</a></li>
+            <li><a href="lists.html">Lists</a></li>
+            <li><a href="grid.html">Grid</a></li>
+    
+        </ul>   
+    </aside>
+
+    <div class="main">
+        <div class="description">
+            <p1>Avatars are images that users can set as their profile picture.They can be custom photos, uploaded by users, or generated as Identicons as a placeholder.
+            </p1>
+        </div>
+
+
+        <div class="avatar-container-size">
+            <p>Avatars with different sizes</p>    
+            <div class="avatar-img">
+                <img class="avatar avatar-sm" src="/Images/1.jpg" alt="" >
```

Comment:
Don't keep alt attribute of images as empty. Better practice to have alt related to image description for better accessibility.

Reply:
> Don't keep alt attribute of images as empty. Better practice to have alt related to image description for better accessibility.

thanks for your suggestion, I will definitely do this